deps/build.jl

```diff
@@ -0,0 +1,28 @@
+using BinDeps
+using Compat
+
+@BinDeps.setup
+
+libmysqlclient = library_dependency("libmysqlclient",
+                                    alias=["libmysqlclient.so",
```

Comment:
How did you try to use unpacked_dir? You tell it where the binaries that you have extract to, then bindeps will look there.